### ISSUE TITLE
Improve example and documentation

### ISSUE THREAD

**Issue by Makio64** [cloudbit-interactive/GSAP-RN#2]
Thanks to bringing gsap to reactnative, its refreshing compare to the other animation solution on rn.

I was wondering if you can push more little demo and documentation about different user case : 
- animation on different Properties like backgroundColor, opacity
- animation on different Elements like View, Text, etc ..
- write if there is any restriction or important performance point to considerate

Thanks in advance :)

**Comment by tufik2** [comment]
Yes I am thinking to do a separate ReactNative App with examples
